spec/features/case_management/invitations_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 RSpec.feature "Sending and accepting invitations" do
   context "As an authenticated user" do
-    let(:user) { create :user_with_org, role: "agent" }
```

Comment:
I think users should have orgs by default, so we should maybe just remove this subfactory and give orgs to all user factories.

Reply:
Yeah good idea! We should that in another PR